FeedStoreChallenge/ManagedObjects/ManagedCache+CoreDataClass.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ManagedCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Vita Solomina on 2021-05-01.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+public class ManagedCache: NSManagedObject {}
```

Comment:
This class is not going to be overridden thus it can be `final`.
Also it is a `CoreDataFeedStore` implementation detail so it should not be public

Please make it `final` and `internal`.
```suggestion
final class ManagedCache: NSManagedObject {}
```